test/functional/wallet_upgradewallet.py

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env python3
+# Copyright (c) 2018-2020 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""upgradewallet RPC functional test
+
+Test upgradewallet RPC. Download v0.15.2 v0.16.3 node binaries:
```

Comment:
18774: nit: This test should be dashified sometimes later, for now it's just skipped so it's ok-ish to have it as is probably.

Reply:
Thanks , so just having as it as is